public/scripts/timeline.js

```diff
@@ -1597,6 +1608,22 @@ function isValidDate(dateString)
     return day > 0 && day <= monthLength[month];
 };
 
+function generateMementoURIList(object)
+{
+    var URIM = [];
+
+    $.each(object,function(index,obj){
+        if($(obj.event_html).attr("src").indexOf("notcaptured") < 0){
+            URIM[index] = obj.event_link;
+        }
+    });
+    var download = document.getElementById("downloadMementoURI");
+    download.href = "";
+    var data = URIM.toString();
+    data = data.replace(/,/g,"\n");
+    download.setAttribute('href', 'data:text/plaincharset=utf-8,' + encodeURIComponent(data));

```

Comment:
You are missing a semicolon here:

```suggestion
    download.setAttribute('href', 'data:text/plain;charset=utf-8,' + encodeURIComponent(data));
```